controls/3.6/def.cf

```diff
@@ -88,6 +88,9 @@ bundle common def
                            # Assume /16 LAN clients to start with
                            "$(sys.policy_hub)/16",
 
+                           # Uncomment below if HA is used
+                           #"@(def.policy_servers)"
```

Comment:
Is it possible to leave this uncommented? At worst, this should end up as an empty list, which should not affect the ACL negatively.